CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# 97.0.1
+
+This hotfix patch release fixes the podspec. The previous release forgot to remove deleted references to card themers.
```

Comment:
Nit: I think we can just say what this release does and remove the _"The previous release forgot to remove deleted references to card themers"_